CONFIGURATION.md

```diff
@@ -281,6 +281,12 @@ validate_additional_rrs:
 [ preferred_ip_protocol: <string> ]
 [ ip_protocol_fallback: <boolean> | default = true ]
 
+# gRPC client metadata (use lowercase for keys)
```

Comment:
I don't see metadata being lowercase a hard requirement by gRPC but maybe I missed, please share docs that mention this?

also add a config check and give error to users or call toLower on keys if lowercase is a hard requirement in gRPC.

Reply:
I don't remember why I thought it was required to be lowercase, the grpc library is already calling toLower. I've removed that phrase from the doc